charts/backbeat/templates/ingestion/consumer_deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "backbeat.fullname" . }}-ingestion-consumer
+  labels:
+    app: {{ template "backbeat.name" . }}-ingestion
+    chart: {{ template "backbeat.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.ingestion.consumer.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat.name" . }}-ingestion
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ template "backbeat.fullname" . }}-consumer
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/usr/src/app/docker-entrypoint.sh"]
+          args: ["npm", "run", "ingestion_consumer"]
+          env:
+            - name: REMOTE_MANAGEMENT_DISABLE
+              value: "{{- if .Values.orbit.enabled }}0{{- else }}1{{- end }}"
+            - name: ZOOKEEPER_AUTO_CREATE_NAMESPACE
```

Comment:
don't think we need zookeeper for the consumer. @jonathan-gramain

Reply:
note sure, in doubt we can leave it